RC1 release notes (#35)
NethServer/ns8-docs: release_notes.rst

center page as usual. After the core components are up-to-date, run the
following manual procedures to complete the upgrade.

- **Core upgrade procedure** -- To upgrade Beta 2 installations run the
  following command on the leader node. It defines the new ``tunadm``
  authorization role, available on new installations since core version
  2.1.0: ::

    redis-cli --raw hvals cluster/module_node | sort -n | uniq | xargs -I NODE_ID -- redis-cli SADD node/NODE_ID/roles/tunadm add-tun remove-tun add-public-service remove-public-service add-custom-zone remove-custom-zone

  For each cluster node, enable the local WebDAV service for backups: ::

    systemctl enable --now rclone-webdav.service

  Finally, only for Rocky Linux nodes, enable the NethServer default
  repositories: ::

    cp -v /etc/nethserver/nethserver.repo /etc/yum.repos.d/nethserver.repo
    dnf config-manager --save --set-disabled appstream baseos extras

- **Password policy** -- Added a new configuration option to the ``Domains
  and users`` page. It is possible to modify the password policy of Samba
  and OpenLDAP domains. Beta 2 installations with OpenLDAP domains require

>   to run a manual procedure to enable the password policy. The the upgrade

Reviewer: ```suggestion
  to run a manual procedure to enable the password policy. The upgrade
```

  procedure is detailed in the next notes. See also
  :ref:`password-policy-section`.

- **User portal** -- Users of a domain can now access a web page to change

>   their password. The user portal is available at

Reviewer: ```suggestion
  their own password. The user portal is available at
```

- List of `known bugs <https://github.com/NethServer/dev/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Abug>`_
- Discussions around `possible bugs <http://community.nethserver.org/c/bug>`_

Major changes on 2023-11-xx
===========================

**Release Candidate 1**

New features introduced by RC1 are:

- **Password policy** -- Added a new configuration option to the ``Domains

>   and users`` page. It is possible to modify the password policy of Samba

Reviewer: ```suggestion
  and users`` page. It is possible to modify the password complexity and expiration policies of Samba
```
